Samples/Client/DirectxWin32/webrtcConfig.json

```diff
@@ -3,9 +3,14 @@
   "turnServer": {
     "uri": "turn:turnserveruri:5349",
     "username": "username",
-    "password": "password"
+    "password": "password",
+    "provider":  ""
```

Comment:
Should we also add the empty **provider** node to the one for native server plugin or just remove it for both?

Reply:
i vote remove it..what do you think?

Reply:
Yep.

Reply:
i did this.